### ISSUE TITLE
Feature Request: Add eversense integration

### ISSUE THREAD

**Issue by timmy354321** [Rytiggy/Glance#86]
I've been doing a bit of looking around and it looks like there aren't any watch faces for an Ionic that read eversense cloud data. Is this something that could be easily added? Eversense has an apple watch applet but i'm hoping to avoid that. Thanks!

**Comment by Rytiggy** [comment]
Totally something I can add to Glance. But I was ll need an API to test off of. Could I use yours?

**Comment by timmy354321** [comment]
Absolutely! I am scheduled for my implantation next Thursday. How should I send you this information? From what I've heard from the eversense people their share platform should be very much like the dexcom share platform.

**Comment by Rytiggy** [comment]
Awesome! Could you join Glance's discord server that's where I post all my pre-release updates and do debugging on the pre-release 

https://discord.gg/RUa7U6F

Once you join I'll add you to the pre-release channel just jet me know when you join
